types/react-select/src/components/Input.d.ts

```diff
@@ -3,18 +3,20 @@ import { CSSObject } from '@emotion/serialize';
 
 import { colors, spacing } from '../theme';
 
-import { PropsWithStyles, ClassNamesState } from '../types';
+import { ClassNamesState, Theme } from '../types';
 
-export type InputProps = PropsWithStyles & {
+export interface InputProps {
     cx: (a: string | null, b: ClassNamesState, c: string) => string | void;
+    getStyles: (name: string, props: any) => {};
```

Comment:
```suggestion
    getStyles: (name: string, props?: any) => {};
```

I feel like I've used `getStyles` without the 2nd arg before. Also, can `{}` be improved?

Reply:
Might also be worth a shared type:

`type GetStylesFn = (name: string, props?: any) => CSSObject`

Reply:
This is just copy-pasted from [place I moved it from](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/50924/files#diff-14ada9224e93f24b2fe4e7de0789b5df415ad44d45a716f121a1a7d3e8f70445L40). It can certainly be improved, but I think that can be a separate PR. I believe the `props` must always be provided.